ansible/configs/sap-hana/default_vars.yml

```diff
@@ -303,7 +303,7 @@ ansible_tower:
     license_file: "/tmp/license.json"
   inventories:
     - name: "sap-hosts"
-      variables: '---\nsap_preconfigure_modify_etc_hosts: true\nsap_domain: \"labs.local\"\nsap_hostagent_installation_type: \"rpm\"\nsap_hostagent_rpm_remote_path: \"/software/SAPHOSTAGENT\"\nsap_hostagent_rpm_file_name: \"saphostagentrpm_44-20009394.rpm\"'
+      variables: '---\nsap_preconfigure_modify_etc_hosts: true\nsap_preconfigure_fail_if_reboot_required: \"no\"\nsap_domain: \"labs.local\"\nsap_hostagent_installation_type: \"rpm\"\nsap_hostagent_rpm_remote_path: \"/software/SAPHOSTAGENT\"\nsap_hostagent_rpm_file_name: \"saphostagentrpm_44-20009394.rpm\"'
```

Comment:
@makentenza Is it possible to make this a little more readable and multiline?

```
variables: >
or 
variables: >-
```

Reply:
@stencell, yes, I could move that to a more readable format, but this is exactly the format you'll get when querying Tower's API, so for simplicity and to allow "copy-paste" from that output, I'll maintain the format as-is now.